src/style-spec.ts

```diff
@@ -69,20 +69,40 @@ import latest from './reference/latest';
 import format from './format';
 import migrate from './migrate';
 import derefLayers from './deref';
-import diff from './diff';
+import diff, {operations as diffOperations} from './diff';
 import ValidationError from './error/validation_error';
 import ParsingError from './error/parsing_error';
-import {StyleExpression, isExpression, createExpression, createPropertyExpression, normalizePropertyExpression, ZoomConstantExpression, ZoomDependentExpression, StylePropertyFunction} from './expression';
+import {FeatureState, StyleExpression, isExpression, createExpression, createPropertyExpression, normalizePropertyExpression, ZoomConstantExpression, ZoomDependentExpression, StylePropertyFunction, Feature, GlobalProperties, SourceExpression, CompositeExpression, StylePropertyExpression} from './expression';
 import featureFilter, {isExpressionFilter} from './feature_filter';
 
 import convertFilter from './feature_filter/convert';
 import Color from './util/color';
 import Padding from './util/padding';
+import Formatted, {FormattedSection} from './expression/types/formatted';
 import {createFunction, isFunction} from './function';
 import convertFunction from './function/convert';
 import {eachSource, eachLayer, eachProperty} from './visit';
-
+import ResolvedImage from './expression/types/resolved_image';
 import validate from './validate_style';
+import {supportsPropertyExpression} from './util/properties';
+import {number} from './util/interpolate';
```

Comment:
I think `number` is a reserved word...?
Can we call this class/method differently?

Reply:
I have renamed it to range()